rkt/run.go

```diff
@@ -313,3 +313,18 @@ func (e *envMap) Strings() []string {
 	}
 	return env
 }
+
+// privateNetList implements the flag.Value interface to allow specification
+// of -private-net with and without values
+type privateNetList string
```

Comment:
What validation criteria do you have in mind? flag.Parse() has already cut off possible whitespaces and has transformed the no-value case to the `true`-string.
That leaves the splitting of the strings with the `,`-delimiter but it doesn't necessarily have to contain a that to be valid.

Do you just want me to move the splitting to the `privateNetList` type?